src/config/config.cc

```diff
@@ -190,6 +188,7 @@ Config::Config() {
        new IntField(&force_compact_file_min_deleted_percentage, 10, 1, 100)},
       {"db-name", true, new StringField(&db_name, "change.me.db")},
       {"dir", true, new StringField(&dir, kDefaultDir)},
+      {"snapshot-dir", false, new StringField(&snapshot_dir, "")},
```

Comment:
```suggestion
      {"snapshot-dir", true, new StringField(&snapshot_dir, "")},
```
I think it should be read-only.